lib/nexpose/util.rb

```diff
@@ -50,7 +50,7 @@ module HostOrIP
     # @return [IPRange|HostName] Valid class, if it can be converted.
     #
     def convert(asset)
-      ips = asset.split(' - ')
+      ips = asset.split('-').map(&:strip)
```

Comment:
I guess we could just to throw a different error. I think this can only fail if the `asset` input is nil, which is true today anyway.